### PR TITLE
reorder a few pages.  no redirects needed.

### PR DESCRIPTION
**Pull request by sedkis**

### CODE REVIEW

File: tyk-docs/content/planning-for-production/redis-mongodb-sizing.md [hunk 1]
```diff
@@ -1,20 +1,11 @@
 ---
-title: Redis and MongoDB Sizing Guide
```
Comment (sedkis) [inline tyk-docs/content/planning-for-production/redis-mongodb-sizing.md:2]: Made "redis & mongo sizing" a child of Redis and mongodb, which contains more generic information about the two

File: tyk-docs/content/planning-for-production/redis-mongodb-sizing.md [hunk 2]
```diff
@@ -1,20 +1,11 @@
...
+title: Redis and Mongo Sizing
 menu:
   main:
-    parent: "Planning for Production"
+    parent: "Redis and MongoDB"
 weight: 2 
 ---
 
-## <a name="analytics"></a>Analytics
```
Comment (sedkis) [inline tyk-docs/content/planning-for-production/redis-mongodb-sizing.md:9]: moved this to /docs/analytics-and-reporting/

File: tyk-docs/content/planning-for-production/redis-mongodb.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+---
+title: Redis and MongoDB
+menu:
+  main:
+    parent: "Planning for Production"
+weight: 2 
+---
+
```
Comment (sedkis) [inline tyk-docs/content/planning-for-production/redis-mongodb.md:8]: all this information (that relates to redis and mongo) was moved from planning and production to its own page

File: tyk-docs/content/tyk-configuration-reference/tyk-pump-configuration.md [hunk 4]
```diff
@@ -418,29 +418,8 @@ From Tyk Pump v0.6.0 you can now create multiple pumps of the same type by by se
 
 ### Capping analytics data
 
-Tyk Gateways can generate a lot of analytics data. A guideline is that for every 3 million requests that your Gateway processes it will generate roughly 1GB of data.
+Tyk Gateways can generate a lot of analytics data. Be sure to read about [capping your Dashboard analytics](/docs/analytics-and-reporting/capping-analytics-data-storage/)
 
-If you have Tyk Pump set up with the aggregate pump as well as the regular MongoDB pump, then you can make the `tyk_analytics` collection a [capped collection](https://docs.mongodb.com/manual/core/capped-collections/). Capping a collection guarantees that analytics data is rolling within a size limit, acting like a FIFO buffer which means that when it reaches a specific size, instead of continuing to grow, it will replace old records with new ones.
```
Comment (sedkis) [inline tyk-docs/content/tyk-configuration-reference/tyk-pump-configuration.md:423]: was duplicate info, just replaced with a link to the analytics page

File: tyk-docs/content/planning-for-production/planning-for-production.mmark [hunk 5]
```diff
@@ -73,21 +73,6 @@ To secure your Tyk installation, you can configure the following settings in you
 
 If you change this values, you need to update the two fields in the dashboard conf file `tyk_analytics.conf`, `tyk_api_config.Host`  and `tyk_api_config.Port`
```
Comment (sedkis) [inline tyk-docs/content/planning-for-production/planning-for-production.mmark:74]: moved all this info that is redis and mongo to its own page under Planning for Production

File: tyk-docs/content/analytics-and-reporting/analytics-and-reporting.md [hunk 6]
```diff
@@ -7,3 +7,10 @@ url: "/analytics-and-reporting"
 ---
 
 The Tyk Dashboard has a full set of analytics functions and graphs that you can use to segment and view your API traffic and activity. The Dashboard offers a great way for you to debug your APIs and quickly pin down where errors might be cropping up and for what clients.
+
```
Comment (sedkis) [inline tyk-docs/content/analytics-and-reporting/analytics-and-reporting.md:10]: moved from another section

File: tyk-docs/content/analytics-and-reporting/capping-analytics-data-storage.md [hunk 7]
```diff
@@ -7,6 +7,31 @@ menu:
 weight: 9 
 ---
 
+Tyk Gateways can generate a lot of analytics data. A guideline is that for every 3 million requests that your Gateway processes it will generate roughly 1GB of data.
```
Comment (sedkis) [inline tyk-docs/content/analytics-and-reporting/capping-analytics-data-storage.md:10]: moved from another section

**Review by marksou — APPROVED** [review]
:thumbsup:

**Pull request merged**